tensorflow/core/kernels/mkl_fused_ops_test.cc

```diff
@@ -310,6 +321,30 @@ TYPED_TEST_P(MklFusedConv2DWithBiasOpTest, SpatialConvolutionAndRelu) {
   this->VerifyFusedConv2D(filter_size, filter_count, {"BiasAdd", "Relu"});
 }
 
+TYPED_TEST_P(MklFusedConv2DWithBiasOpTest, OneByOneConvolutionAndRelu6) {
+  const int filter_size = 1;
```

Comment:
Per our [Style Guide](https://google.github.io/styleguide/cppguide.html#Constant_Names) on naming constants,
```suggestion
  const int kFilterSize = 1;
```
Please change other places as well.